quinn-proto/src/endpoint.rs

```diff
@@ -126,6 +127,7 @@ impl Endpoint {
         local_ip: Option<IpAddr>,
         ecn: Option<EcnCodepoint>,
         data: BytesMut,
+        rng: impl FnOnce() -> StdRng,
```

Comment:
Why doesn't this and below use `self.rng`?

Reply:
I got the impression that the endpoint and connection it creates are using freshly seeded rngs, as they were calling `StdRng::from_entropy` separately. Should I use one to seed the other in the upcoming fix?

Reply:
Yes, I think that would make sense.

Reply:
Yeah, using fresh entropy in each was a matter of convenience, not a principled requirement. Using well-conditioned cryptographic RNGs to seed eachother is robust.